### PR TITLE
Datalake2Sentinel Connector

### PR DESCRIPTION
**Pull request by cert-orangecyberdefense**
The Datalake to Microsoft Sentinel integration allows you to upload indicators from Orange Cyberdefense **Datalake** platform to Microsoft Sentinel.

New data connector for integration Datalake to Sentinel

* JSON file with data connector definition
* Short README with link to GitHub repository where integration code is located

### CODE REVIEW

**Comment by cert-orangecyberdefense** [comment]
@microsoft-github-policy-service agree company="Orange Cyberdefense"

**Comment by v-prasadboke** [comment]
Hello @cert-orangecyberdefense for SVG logo please follow below guidelines, I tried to correct the logo but was facing issue. 

1. Logo needs to be in SVG format and under 5 Kb
2. Ensure raw file of logo does not have any of the following:
> - cls and style formats
> - embedded png formats
> - xmlns:xlink

3.  Logo needs to be in SVG format and under 5 Kb
4.  Logo scales well to fit in a 75 px square
5.  SVG code file is formatted well for readability

Also for Solution, you need to add input file and solutionmetadata which is required to repackage solution. Please refer to any of the solution for more clarification. Input file is located inside data folder of the solution.

**Comment by cert-orangecyberdefense** [comment]
> Hello @cert-orangecyberdefense for SVG logo please follow below guidelines, I tried to correct the logo but was facing issue.
> 
>     1. Logo needs to be in SVG format and under 5 Kb
> 
>     2. Ensure raw file of logo does not have any of the following:
> 
> 
> > * cls and style formats
> > * embedded png formats
> > * xmlns:xlink
> 
>     3. Logo needs to be in SVG format and under 5 Kb
> 
>     4. Logo scales well to fit in a 75 px square
> 
>     5. SVG code file is formatted well for readability
> 
> 
> Also for Solution, you need to add input file and solutionmetadata which is required to repackage solution. Please refer to any of the solution for more clarification. Input file is located inside data folder of the solution.

Hello @v-prasadboke thank you for your reply. l have added the require files and will soon provide a new logo.

File: Solutions/Datalake2Sentinel/SolutionMetadata.json
Comment (v-prasadboke) [inline Solutions/Datalake2Sentinel/SolutionMetadata.json:1]: Can you please once check and confirm on the Publisherid and Support fields

File: Solutions/Datalake2Sentinel/Data/Solution_Datalake2Sentinel.json
Comment (v-prasadboke) [inline Solutions/Datalake2Sentinel/Data/Solution_Datalake2Sentinel.json:5]: Also please add a few more lines to the description

Comment (cert-orangecyberdefense) [reply Solutions/Datalake2Sentinel/Data/Solution_Datalake2Sentinel.json]: Ok

**Comment by v-prasadboke** [comment]
Thanks for commiting the required changes. Will take a look at it once.

File: Solutions/Datalake2Sentinel/SolutionMetadata.json
Comment (v-prasadboke) [inline Solutions/Datalake2Sentinel/SolutionMetadata.json:2]: Hello @cert-orangecyberdefense, Please confirm on publisher id once

Comment (v-prasadboke) [reply Solutions/Datalake2Sentinel/SolutionMetadata.json:2]: Will Microsoft team publish it

Comment (cert-orangecyberdefense) [reply Solutions/Datalake2Sentinel/SolutionMetadata.json:2]: Hello @v-prasadboke thanks for the feedback, let us check it and we will get back to you

**Comment by v-prasadboke** [comment]
Hello @cert-orangecyberdefense, Waiting for reply on Publisher ID

**Comment by cert-orangecyberdefense** [comment]
> Hello @cert-orangecyberdefense, Waiting for reply on Publisher ID

Hi @v-prasadboke the update of the publisher ID is now ok, waiting for your approval to continue the process
Thanks

**Review by v-prasadboke — DISMISSED** [review]

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**
